picard/mbjson.py

```diff
@@ -248,9 +248,12 @@ def country_list_from_node(node):
         country = []
         for release_event in node['release-events']:
             try:
-                country.append(release_event['area']['iso-3166-1-codes'][0])
-            except KeyError:
+                country_code = release_event['area']['iso-3166-1-codes'][0]
+            # TypeError in case object is None
+            except (KeyError, IndexError, TypeError):
                 pass
+            if country_code:
```

Comment:
This should be in the try/except block after `country_code = ...` else it may be undefined